Modify S5542: Learn-As-You-Code migration (#2011)
SonarSource/rspec: rules/S5542/summary.adoc

> This vulnerability exposes encrypted data to an unnecessarily large number of

Reviewer: ```suggestion
This vulnerability exposes encrypted data to a number of
```

"unnecessarily large" feels imprecise as there only has to be one successful attack.
